core/src/main/scala/kafka/server/ConfigHandler.scala

```diff
@@ -63,6 +64,29 @@ class TopicConfigHandler(private val logManager: LogManager, kafkaConfig: KafkaC
       val logConfig = LogConfig(props)
       logs.foreach(_.config = logConfig)
     }
+
+    val brokerId: Int = kafkaConfig.brokerId
```

Comment:
Nitpick: we don't usually include the type annotation for simple local variables like this.